tbx/static_src/sass/components/_footer.scss

```
@@ -2,11 +2,66 @@
 
 .footer {
     $root: &;
-    background-color: var(--color--light-grey);
-    padding: ($grid * 2) $variable-gutter--small;
 
-    @include media-query(menu-break) {
-        padding: ($grid * 3) 3.5vw;
+    &__container {
+        background-color: var(--color--light-grey);
+        padding: ($grid * 2) $variable-gutter--small;
+
+        @include media-query(menu-break) {
+            padding: ($grid * 3) 3.5vw;
+        }
+    }
+
+    &__logos {
+        $root: &;
```

We've accidentally defined the `$root` variable twice here, if you get rid of this one the hover effect will work as expected

```suggestion
        $root: &;
```